api/customAttributes.js

```diff
@@ -92,6 +92,13 @@ module.exports = function(app) {
     else if (dimension === "CIP") {
       retObj.stem = id.length === 6 ? stems.includes(id) ? "Stem Major" : false : "Contains Stem Majors"
     }
+    else if (dimension === "University") {
+      const opeid = await axios.get(`${origin}/api/university/opeid/${id}`)
+        .then(resp => resp.data)
+        .catch(() => []);
+
+      retObj.opeid = opeid && opeid.opeid ? opeid.opeid : false;
```

Comment:
No need to do an API call here, this can come directly from the `app.settings.cache` above. There's a lookup object called `opeid`, which the endpoint you're using uses as a pass-through: https://github.com/DataUSA/datausa-site/blob/opeid/api/crosswalks.js#L317

So this logic could just be:
```js
retObj.opeid = opeid[id];
```